.github/workflows/graphql-api-ci.yml

```diff
@@ -35,6 +36,6 @@ jobs:
       - name: Run ESLint
         run: pnpm eslint graphql-api
       - name: Type check
-        run: pnpm ts-node ./browser/build/buildHelp.ts && pnpm typecheck
+        run: pnpm ts-node ./browser/build/buildHelp.ts && cd graphql-api && pnpm tsc --noEmit --project ../tsconfig.json
```

Comment:
The typecheck actually should run on both codebases if either are changed, since IIRC there are some types defined in one and used in the other, so we want to make sure those remain consistent.

Reply:
Aha, ok, thanks for the catch!